labs/ipfs/how-to/create-volume.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: How to create a volume
+  description: This page explains how to create a volume using Scaleway IPFS Pinning
+content:
+  h1: How to create a volume
+  paragraph: This page explains how to create a volume using Scaleway IPFS Pinning
+tags: ipfs-pinning ipfs-storage ipfs pinning storage labs web3 volume
+dates:
+  validation: 2023-06-27
+  posted: 2023-06-27
+categories:
+  - labs
+---
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+</Message>
+
+1. Click **IPFS Pinning** in the **Labs** section of the [Scaleway Console](https://console.scaleway.com) side menu. The IPFS Pinning section displays.
+2. Click **Create a Volume** to create a volume, which is the storage area for a set of pins.
```

Comment:
I think it would be better to create a concept for "volume", explain what it is there and link to it here, wdyt?

Reply:
I think that's a great idea!